sway-lib-std/src/context/call_frames.sw

```diff
@@ -5,6 +5,14 @@ library call_frames;
 
 use ::contract_id::ContractId;
 
+// Note that everything when serialized is padded to word length.
+//
+// Call Frame        :  saved registers = 8*WORD_SIZE = 8*8 = 64
+// Reserved Registers:  frame pointer   = 6*WORD_SIZE = 6*8 = 48
+
+const SAVED_REGISTERS_OFFSET = 64;
+const CALL_FRAME_OFFSET = 48;
```

Comment:
```suggestion
const PREV_FRAME_POINTER_OFFSET = 48;
```

Reply:
Please update this constant's name throughout.

Reply:
Will do. Clicking the "commit suggestion" button makes it too easy to miss this stuff.